### ISSUE TITLE
Change "Cargo Doc" section to "Rustdoc" on tools page

### ISSUE THREAD

**Issue by camelid** [rust-lang/www.rust-lang.org#1255]
### What needs to be fixed?
Currently on the tools page, it says:

![image](https://user-images.githubusercontent.com/37223377/91777385-8498ed00-eba4-11ea-86e6-701d3ba18f9f.png)

"Cargo Doc" seems a bit out of place given that another section is shown as "Rustfmt", when usually you use it as `cargo fmt`.

### Page(s) Affected
/tools

### Suggested Improvement
Change "Cargo Doc" to "Rustdoc"

### Side Note
Also add `cargo fmt` to this list on /learn/getting-started?

![image](https://user-images.githubusercontent.com/37223377/91777528-f07b5580-eba4-11ea-83cf-c596f81213b0.png)

**Comment by Manishearth** [comment]
I'm not sure I agree, the name `rustdoc` is not as popular in the rust community since people never directly invoke it, whereas people are kinda used to the names rustfmt and clippy.

**Comment by camelid** [comment]
But this page is mostly for new contributors who won't necessarily know the names clippy and rustfmt anyway, right? That's why it explains what they do. And calling it Rustdoc will probably get more people more familiar with it.

**Comment by Manishearth** [comment]
@camelid Yes, but the point is that you want to teach people the terms other people will use, so that they do not have to learn a second thing.
